src/bin/ion/commands/inspect.rs

```diff
@@ -223,6 +223,8 @@ struct IonInspector<'a, 'b> {
     skip_complete: bool,
     limit_bytes: usize,
     hide_expansion: bool,
+    ephemeral_depth: usize,
+    line_has_comment: bool,
```

Comment:
🪧 As you'll see later, the inspector now tracks whether a comment has already been opened on the current line of output. This allows multiple bits of "commentor" code to add comments without coordination.